pandas/core/frame.py

```diff
@@ -8162,12 +8171,20 @@ def corr(self, method="pearson", min_periods=1) -> "DataFrame":
               dogs  cats
         dogs   1.0   0.3
         cats   0.3   1.0
+        >>> df.T.corr(method=histogram_intersection, axis=1)
+              dogs  cats
+        dogs   1.0   0.3
+        cats   0.3   1.0
```

Comment:
Can you use an example that does the correlation across rows, rather than taking a transpose twice?